go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/bmatcuk/doublestar/v4 v4.4.0 // indirect
```

Comment:
do you know from where this dep is coming from?

Reply:
removed